vendor.conf

```diff
@@ -0,0 +1,21 @@
+github.com/Azure/go-ansiterm d6e3b3328b783f23731bc4d058875b0371ff8109
+github.com/containerd/continuity 004b46473808b3e7a4a3049c20e4376c91eb966d
+github.com/docker/docker 86f080cff0914e9694068ed78d503701667c4c00
```

Comment:
OK, this is likely a question that will lead to Go learning for me.   How did you pick this commit?  It seems to be a couple years old now, is that the one that was in play currently prior to adding vendor.conf?